src/main/java/seedu/duke/Duke.java

```diff
@@ -57,6 +57,7 @@ private void runCommandLoop() {
             String result = executeCommand(command);
             interfaceManager.prints(result);
         } while (!ExitCommand.isExit(command));
+        interfaceManager.prints(Formatter.getGoodbyeMessage());
```

Comment:
Are you moving all the message to a class? If so you can call the format function inside ExitCommand execute so that there is no need for an additional call here